Sources/TransferMethod/AddTransferMethodViewController.swift

```diff
@@ -321,7 +321,8 @@ extension AddTransferMethodViewController: AddTransferMethodView {
         }
     }
 
-    func showTransferMethodFields(_ fields: [HyperwalletField], _ transferMethodTypeDetail: TransferMethodTypeDetail) {
+    func showTransferMethodFields(_ fields: [HyperwalletFieldGroup],
+                                  _ transferMethodTypeDetail: TransferMethodTypeDetail) {
```

Comment:
This should be HyperwalletTransferMethodType

Reply:
Create extension of HyperwalletTransferMethodType for formatFeesProcessingTime()

Reply:
fixed